### PR TITLE
REFACTOR: simulation setup part 1

### PR DESCRIPTION
**Pull request by ring630**
This PR only moves the files.

### CODE REVIEW

File: tests/legacy/system/test_edb.py [hunk 1]
```diff
@@ -1028,6 +1028,25 @@ def test_hfss_simulation_setup(self):
...
+        edbapp = edb_examples.get_si_verse()
+        setup1 = edbapp.create_hfss_setup("setup1")
+        """sweep1 = setup1.add_frequency_sweep(name="sweep1",
+                                   frequency_sweep=[
+                                   ["linear count", "1MHz", "10MHz", 10],
+                                   ])
+        sweep2 = setup1.add_frequency_sweep(name="sweep2",
+                                            frequency_sweep=[
+                                                ["log scale", "1kHz", "100kHz", 10],
+                                            ])"""
```
Comment (SMoraisAnsys) [inline tests/legacy/system/test_edb.py:1041]:
```suggestion
        sweep1 = setup1.add_frequency_sweep(name="sweep1",
                                   frequency_sweep=[
                                   ["linear count", "1MHz", "10MHz", 10],
                                   ])
        sweep2 = setup1.add_frequency_sweep(name="sweep2",
                                            frequency_sweep=[
                                                ["log scale", "1kHz", "100kHz", 10],
                                            ])
```
Why is this part of the code "commented" ("""...""") ?
Also, could you add assert calls to see if everything worked fine ?

File: tests/legacy/system/test_edb_config_json.py [hunk 2]
```diff
@@ -60,6 +60,48 @@ def test_01_create_edb(self, edb_examples):
...
+                        },
+                    ],
+                },
+            ]
+        }
+        edbapp = edb_examples.get_si_verse()
+        assert edbapp.configuration.load(data, apply_file=True)
+        # assert len(edbapp.setups["hfss_setup_1"].frequency_sweeps["sweep1"].frequencies) == 16
+        # assert len(edbapp.setups["hfss_setup_1"].frequency_sweeps["sweep2"].frequencies) == 20
+        # assert len(edbapp.setups["hfss_setup_1"].frequency_sweeps["sweep3"].frequencies) == 11
```
Comment (SMoraisAnsys) [inline tests/legacy/system/test_edb_config_json.py:102]: Should we remove this comments or uncomment the lines of code ?

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
I left minor proposal changes because I felt that something might have been forgotten in the tests refactoring. Good job otherwise !

**Review by svandenb-dev — DISMISSED** [review]
LGTM

File: tests/legacy/system/test_edb_config_json.py (hunk 2 above)
Comment (svandenb-dev) [reply tests/legacy/system/test_edb_config_json.py:102]: Yes I agree this should be removed

Comment (svandenb-dev) [reply tests/legacy/system/test_edb_config_json.py]: done

**Review by SMoraisAnsys — DISMISSED** [review]

**Review by svandenb-dev — APPROVED** [review]
LGTM

**Pull request merged**